util/src/main/scala/scala/scalanative/util/ShowBuilder.scala

```diff
@@ -31,14 +31,26 @@ final class ShowBuilder {
     indentation -= n
 
   def newline(): Unit = {
-    sb.append("\n")
-    sb.append("  " * indentation)
+    out.append("\n")
+    out.append("  " * indentation)
   }
 
   def clear(): Unit = {
```

Comment:
This method does not make sense on `ShowBuilder` anymore, since it cannot be correctly implemented in `FileShowBuilder`. If it is necessary, it should be moved to `InMemoryShowBuilder` only.

Reply:
Actually `clear` is no longer needed now, so I've removed it. Thanks for tracking me on this.